easybuild/easyblocks/e/extrae.py

```diff
@@ -79,6 +79,9 @@ def configure_step(self):
         # TODO: make this optional dependencies
         self.cfg.update('configopts', "--without-dyninst")
 
+        # Workaround to make MPI tests pass in some platforms
+        self.cfg.update('configopts', "--disable-instrument-dynamic-memory")
```

Comment:
This is (hopefully) something version specific, so we might be better off putting this in the easyconfig for that version rather than making it permanent here (or at least add a version guard here)

Reply:
You are right. I put it in the easyconfig.